acceptance/bundle/python/resource-ordering/script

```diff
@@ -0,0 +1,3 @@
+trace uv run --with databricks-bundles==0.7.2 -q $CLI bundle validate --output json
```

Comment:
why does this install the package from PyPI and not from local repo?

Reply:
local package doesn't contain all necessary code yet